src/gui/qgsmaptoolcapture.h

```diff
@@ -143,6 +143,7 @@ class GUI_EXPORT QgsMapToolCapture : public QgsMapToolAdvancedDigitizing
     {
       StraightSegments, //!< Default capture mode - capture occurs with straight line segments
       CircularString, //!< Capture in circular strings
+      Streaming, //!< Streaming points digitizing mode (points are automatically added as the mouse cursor moves)
```

Comment:
```suggestion
      Streaming, //!< Streaming points digitizing mode (points are automatically added as the mouse cursor moves; since QGIS 3.20) 
```

Reply:
Done